desktop/src/app/components/certificate/action/add/add-certificate-form.html

```diff
@@ -4,6 +4,12 @@
             [title]="'add-certificate-form.certificate.label' | i18n"
             [subtitle]="'add-certificate-form.certificate.description' | i18n">
             <div class="form-element">
+                <bl-banner class="certificate-deprecation-warning" type="warning">
+                    <div message>
+                        The Batch account certificates feature is deprecated and will be retired on February 29, 2024. Please transition to using Azure Key Vault to securely access and install certificates on your Batch pools.
+                        <a class="certificate-warning-alternative-link" (click)="openLink('https://learn.microsoft.com/en-us/azure/batch/batch-certificate-migration-guide')" href="javascript:void(0)">Learn more</a>
```

Comment:
Links to docs should omit the "/en-us" part of the URL so they can be properly localized.

Reply:
Also "Learn more" could be extracted to common.18n.yml since it's such a commonly used string.

Reply:
Removed "/us-en" from links and added "learn more" to common.i18n.yml